### PR TITLE
logo updated

### PR DESCRIPTION
**Pull request by Rishab42**

### CODE REVIEW

**Comment by Rishab42** [comment]
@Showtim3 Ready to review code

File: src/comp/Home.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ function Home(){
         <div className="home-container">
             <div className="navbar">
                 <div className="logo">
-                    <span>LoGo</span>
+                    <span>LoGoo</span>
```
Comment (Showtim3) [inline src/comp/Home.js]: Dummy change required.

**Review by Showtim3 — CHANGES_REQUESTED** [review]

File: src/comp/Home.js (hunk 1 above)
Comment (Rishab42) [reply src/comp/Home.js]: Commited

**Review by Showtim3 — APPROVED** [review]
LGTM

**Pull request merged**